### DISCUSSION TITLE
Answer to question about jury page

### DISCUSSION THREAD

**Discussion by jflamy-dev** [jflamy/owlcms4#402] [Q&A]
Original question was asked [here](https://github.com/jflamy/owlcms4/discussions/397#discussioncomment-289808)

When using refereeing with a jury, you need two browsers.   Typically, the competition setup is as follows

- Computer 1: runs the athlete facing display, or an attempt board.  This computer has the refereeing devices attached to it, that will emit keystrokes 1 to 6 for the three referees.
- Computer 2: runs the jury console.  This computer also has keypads, but for the jury members.  They also use keystrokes 1 to 6 (3 person jury) or 1 to 0 (5-person jury).  But the keypads on the jury console control only the jury.
- When the browser on computer 1 senses the referee decisions, it sends each to owlcms.  Then owlcms relays the decisions as they happen on the announcer console and on the jury console.

For testing purposes, when you have a single computer,  you can actually use a single browser, BUT with two windows or two tabs.  One will have the athlete facing display (or attempt board, doesn't matter).  When you click in that tab, you are simulating the referees using your keyboard.  If you have several windows open, you will see the decisions on the bottom part of the jury window and on the announcer window.

If you switch focus by clicking on the jury console, you can now use your keyboard to simulate the jury input.  The decisions will appear as green squares until all jurors have entered their decisions (you need to give 1 or 2 AND 3 or 4 AND 5 or 6  -- then the decisions are shown.  This is so the jury members don't influence one another.

**Comment by jonathanuno** [comment]
Mr. Lamy,

I see what the issue is. I thought referee keystrokes were done in the Jury Console page. Referee keystrokes are actually taken in the Attempt Board display window. This was a bit confusing to me since I associated all aspects of controlling the meet with the "Run Lifting Group" pages and not the "Start Displays" pages. Everything is working as expected now. 

Your method actually makes the most sense because the referees and their devices will usually be physically close to the athlete facing display or attempt board. Thank you for thinking of everything and developing such a fantastic system.

**Reply by jflamy-dev** [reply]
Agreed that it can be confusing.  Basically all the black screens seen by the public are under Display, and all the white ones are under "run the meet".

**Reply by jonathanuno** [reply]
Would you consider adding a note to the Refereeing section of the guide to say that referee input is taken in the Attempt Board window?
